server/pages/index.js

```diff
@@ -121,7 +121,7 @@ function getDefaultContext( request ) {
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
 		catchJsErrors: '/calypso/catch-js-errors-' + 'v2' + '.min.js',
-		isPushEnabled: !! config.isEnabled( 'push-notifications' )
+		isPushEnabled: !! config.isEnabled( 'push-notifications' ),
```

Comment:
we should remove `isPushEnabled` entirely, otherwise in environments where only the ab-test exists, we won't have the manifest file